esp32_s3_lcd_ev_board/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+set(SUB_BOARD_SRC "")
+
+if (CONFIG_BSP_LCD_SUB_BOARD_480_480)
+    set(SUB_BOARD_SRC "sub_board/sub_board_480x480.c")
+elseif (CONFIG_BSP_LCD_SUB_BOARD_800_480)
+    set(SUB_BOARD_SRC "sub_board/sub_board_800x480.c")
+endif()
+
+idf_component_register(
+    SRCS "esp32_s3_lcd_ev_board.c" ${SUB_BOARD_SRC}
+    INCLUDE_DIRS "include"
```

Comment:
Could you please add `PRIV_INCLUDE_DIRS priv_include` ? That way we can get rid of the ugly `#include "../priv_include/bsp_err_check.h"`  includes in `sub_board` directory

Reply:
Fixed, PTAL [here](https://github.com/espressif/esp-bsp/pull/89/commits/abc7099df435b938ce2cd02f1ecafc7ca546c79c#diff-f87fc8945850544b13520a5fb8f70dde17fd0ef30f1507472166d6659d5b5e98R12).